### ISSUE TITLE
drawing tablet issue on chrome and obsidian

### ISSUE THREAD

**Issue by hyperphysin** [excalidraw/excalidraw#6592]
The first stroke works fine but when you try to draw the second stroke, it moves the cavas. To draw a stroke again, you need to raise your pen high up and get back to the tablet.
This doesn't happen on firefox and safari.

**Comment by dwelle** [comment]
Hi. So as I understand it, this relates both to Obsidian and excalidraw.com? What tablet are you using?

**Comment by hyperphysin** [comment]
> Hi. So as I understand it, this relates both to Obsidian and excalidraw.com? What tablet are you using?

Yes, obsidian excalidraw plugin and excalidraw.com on chromium based web browsers.
(It works fine on other browsers.)
I'm using virtual tablet(https://www.sunnysidesoft.com/virtualtablet) with galaxy s22 ultra.

**Comment by rslhdyt** [comment]
I have the same issue, I use Brave browser and Veikk drawing tablet.
The first stroke worked as expected, but the cursor moved the canvas in the second stroke and the drawing mode stopped working, even when using a mouse.

**Comment by mantans** [comment]
i have the same issue using obsidian excalidraw plugin + virtual tablet app on android device. I notice that after the first stroke to avoid drawing mode stop working, you must clear the canvas on virtual tablet. This makes the work pipeline really slow.

**Comment by mantans** [comment]
> > Hi. So as I understand it, this relates both to Obsidian and excalidraw.com? What tablet are you using?
> 
> Yes, obsidian excalidraw plugin and excalidraw.com on chromium based web browsers. (It works fine on other browsers.) I'm using virtual tablet(https://www.sunnysidesoft.com/virtualtablet) with galaxy s22 ultra.

On virtualtablet app go to Settings and switch off Mirror mode, everything will work like a charm.

**Comment by vietbui1999ru** [comment]
Hi, is there anyone working on this issue yet? For anyone still experiencing issues with using VirtualTablet and Excalidraw on browsers, my current workaround is switching to other browsers that are not based on Chromium. I have tested on Safari and Firefox, MacOS Sequoia, and my S pen and Samsung Tab S7 work as expected.

**Comment by gmcouto** [comment]
Can confirm: other browsers work. The issue is with Chrome/Chromium
